File: if-else/JavaScript/if-else.ts
function main(n: number) {
	if (!(n % 2)) {
		console.log('Weird');
	} else {
		if (n >= 2 && n <= 5) {
			console.log('Weird');
		}
		if (n >= 6 && n <= 20) {
			console.log('Weird');
		} else {
			console.log('Not Weird');
		}
	}
}
main(3); //Expected Output: 'Weird'
main(24); //Expected Output: 'Not Weird'

File: if-else/Python/if_else.py
"""
Given an integer,n, perform the following conditional actions:
If n is odd, print Weird
If n is even and in the inclusive range of  to , print Not Weird
If n is even and in the inclusive range of  to , print Weird
If n is even and greater than , print Not Weird
"""


def check_n(num):
    if n % 2:
        return ('Weird')
    else:
        if n >= 2 and n <= 5:
            return ('Not Weird')
        elif n >= 6 and n <= 20:
            return ('Weird')
        else:
            return ('Not Weird')


if __name__ == '__main__':
    n = int(input('What number do you want to check?').strip())
    print(check_n(n))
